Library/Tracking/TrackingHelpers.swift

```diff
@@ -11,4 +12,14 @@ final class TrackingHelpers {
       return .fixErroredPledge
     }
   }
+
+  static func discoveryContext(for sort: DiscoveryParams.Sort) -> KSRAnalytics.TypeContext
```

Comment:
Mentioned this in a previous comment - I would like you to try a different approach here. We shouldn't be including a new import for functionality like this. I also think there has to be a cleaner way to map these enum values if they are nearly identical.

Reply:
Okay, I'll think of something.\